docs/api/web-frame.md

```diff
@@ -109,6 +109,43 @@ this limitation.
 
 ### `webFrame.getResourceUsage()`
 
+Returns `Object`:
```

Comment:
Returns an Object describing usage information of Blink's internal memory caches: